.github/workflows/tests.yml

```diff
@@ -3,29 +3,36 @@ name: Test
 on: [push]
 
 jobs:
-  test:
-    runs-on: ubuntu-latest
-    services:
-      mariadb:
-        image: ispyb/ispyb-pydb:latest
-        ports:
-          - 3306:3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
-    strategy:
-      matrix:
-        python-version: ["3.10"]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v3
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
-        run: |
-          sudo apt-get install -y libldap2-dev libsasl2-dev
-          pip install -r requirements.txt
-          pip install -r requirements-dev.txt
-          pip install -e . --no-deps
-      - name: Run tests
-        run: |
-          pytest
+    test:
```

Comment:
I think we have different settings for yaml autoformat in vscode! Now its auto applying formatting on save its conflicting. In my vscode its 2 space. We should enforce this properly in the `vscode/.settings`

Reply:
what autoformatter do you use for yaml? I had prettier which seems to ignore the tab size setting for yaml and use its own setting instead.